src/components/MaText/MaText.stories.js

```diff
@@ -32,9 +38,10 @@ export default {
 
 const Template = (args, { argTypes }) => ({
   props: Object.keys(argTypes),
-  template: `<div style="display: flex;">
+  template: `
+  <div style="width: 20rem">
```

Comment:
I'd keep the flex layout for now, so we can easily see the text without the blueish background

```suggestion
  <div style="display: flex;width: 20rem">
```